omnistrate.enterprise.yaml

```
@@ -36,122 +36,9 @@ x-omnistrate-load-balancer:
-        aggregationFunction: sum
-    redis_net_input_bytes_total:
-      "Total Network Input Bytes":
-        aggregationFunction: sum
-    redis_net_output_bytes_total:
-      "Total Network Output Bytes":
-        aggregationFunction: sum
-    redis_commands_duration_seconds_total:
-      "Total Command Duration (Seconds)":
-        aggregationFunction: sum
-
+    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)